integration/consul_catalog_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	checker "github.com/vdemeester/shakers"
+	check "gopkg.in/check.v1"
+)
+
+// Consul catalog test suites
+type ConsulCatalogSuite struct {
+	BaseSuite
+	client *api.Client
+}
+
+func (s *ConsulCatalogSuite) SetUpSuite(c *check.C) {
+	s.createComposeProject(c, "consul_catalog")
+	s.composeProject.Up()
+
+	config := api.DefaultConfig()
+	config.Address = "127.0.0.1:8500"
```

Comment:
This is most likely not going to work out, because on CI (and when doing `make test-integration`) we are in a container, but pointing to the engine we are running on to start the compose services. So we should add a way to easily get the container information, and mainly the API.

I'll take a look into it really soon (today or tomorrow) to add an easy way to do that :wink:.